python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -26,6 +25,7 @@
 from Ganga.GPIDev.Schema import ComponentItem, FileItem, GangaFileItem, Schema, SimpleItem, Version
 from Ganga.Runtime.spyware import ganga_job_submitted
 from Ganga.Utility.Config import ConfigError, getConfig
+from Ganga.Utility.external.OrderedDict import OrderedDict as oDict
```

Comment:
There was some convention of using `oDict` elsewhere in the codebase. If this is disliked I'll change back but was just trying to keep it the same when we come to make some edit to rely on this.